pgdrive/scene_manager/__init__.py

```diff
@@ -1,3 +1,3 @@
 class TrafficMode:
-    Reborn = 0
-    Add_once = 1
+    Reborn = "reborn"
+    Trigger = "trigger"
```

Comment:
Let's add a compatibility helper in the place that use this term, if necessary.

But it seems that you are removing the "add_once" thing completely? Does this infect the performance of agent as well as the map loading? Do we need to regenerate all maps?

Reply:
No, traffic mode is not recorded in map.json.